modules/flight_interface/flight_interface_worker.py

```diff
@@ -34,8 +39,16 @@ def flight_interface_worker(
 
         time.sleep(period)
 
+        command = None
+
+        try:
+            command: decision_command.DecisionCommand = command_in_queue.queue.get_nowait()
+            interface.run_decision_handler(command)
+        except queue.Empty:
+            pass
+
```

Comment:
maybe move this block to the end so we're guaranteed that the command is not none
1. remove the line initializing command = none
2. instead of passing in the exception, continue instead
3. put the `interface.run_decision_handler(command)` line after the try catch block so the continue will skip it